### PR TITLE
✅ Enable tests for Python 3.11

### PR DESCRIPTION
**Pull request by tiangolo**
✅ Enable tests for Python 3.11

This will have to wait for a Uvloop wheel to be available.

### CODE REVIEW

**Comment by Kludex** [comment]
Do you want for uvloop to be a requirement?

File: tests/test_custom_middleware_exception.py [hunk 1]
```diff
@@ -56,6 +56,12 @@ async def __call__(self, scope, receive, send):
 
 @router.post("/middleware")
 def run_middleware(file: UploadFile = File(..., description="Big File")):
+    # TODO: without this, pytest raises an error, and it causes a segmentation fault on
+    # Python 3.11-rc.2, the app works normally when run with Uvicorn, not sure if it's
+    # a problem with Python 3.11, pytest, or FastAPI.
+    # But FastAPI (or Starlette) should probably try to close these
+    # SpooledTemporaryFiles automatically by default
+    file.file.close()
```
Comment (Kludex) [inline tests/test_custom_middleware_exception.py:64]: I think it's on starlette

**Comment by raphaelauv** [comment]
basic tests are working -> https://github.com/raphaelauv/fastAPI-aiohttp-example/pull/6

File: tests/test_custom_middleware_exception.py (hunk 1 above)
Comment (tiangolo) [reply tests/test_custom_middleware_exception.py]: This was handled here: https://github.com/tiangolo/fastapi/pull/5465 :tada:

**Pull request merged**